src/glewlwyd.c

```diff
@@ -38,6 +38,8 @@
 
 #include "glewlwyd.h"
 
+void* signal_thread(void *arg);
```

Comment:
I prefer either to declare functions in `glewlwyd.h` or implement them above the calling function. In this case I think declaring `signal_thread` in `glewlwyd.h` will be sufficient and in harmony with the code style.